cmd/deploy.go

```diff
@@ -166,7 +178,8 @@ func InitRunCommand() (*cobra.Command, error) {
 	run.Flags().String("url", os.Getenv("KUBERNETES_CLUSTER_URL"),
 		"URL of the Kubernetes cluster. Overrides KUBERNETES_CLUSTER_URL env variable in AppController pod.")
 
-	run.Flags().StringP("replicas", "n", "1", "number of flow replicas: [+]number")
+	run.Flags().StringP("replicas", "n", "", "number of flow replicas: [+|-]number")
+	run.Flags().Bool("delete-external", false, "permit delete of external resources")
```

Comment:
yes, external means resources that are used in dependencies but do not have resdef. I.e. external to the graph. Such resources were not created by AC and shouldn't be deleted by AC unless this flag is specified